templates/seo/_canonical_url.html

```diff
@@ -0,0 +1,3 @@
+{% if path %}
+    <link rel="canonical" href="{{ path }}" />
+{% endif %}
```

Comment:
j'uniformiserais bien un peu le nom des variables, `path` peut faire référence à d'autres choses (surtout quand on manipule des pages dans wagtail) 

```suggestion
{% if canonical_url %}
    <link rel="canonical" href="{{ canonical_url }}" />
{% endif %}
```